src/vma/dev/net_device_table_mgr.cpp

```diff
@@ -253,11 +253,20 @@ int net_device_table_mgr::map_net_devices()
 			continue;
 		}
 
+		int iftype = get_iftype_from_ifname(ifa->ifa_name);
+		if (iftype == ARPHRD_INFINIBAND) {
+			if (!check_ib_device_supported(ifa->ifa_name) || !verify_ipoib_mode(ifa)) {
+				continue;
+			}
+		} else {
+			if (!check_qp_creation(ifa->ifa_name, IBV_QPT_RAW_PACKET)) {
```

Comment:
Not sure we need to print WARNING at all (currently inside the function).
If this is a loopback interface we print in DEBUG and mark as not offloaded, same if this was a Broadcom NIC...
so here to I think we should not print in WARN but only in DEBUG.